cypress/integration/page.spec.js

```diff
@@ -262,7 +291,7 @@ describe('', () => {
       });
 
       it('should not have users hardcoded', () => {
-        cy.intercept('**/users', { fixture: 'someUsers' })
+        cy.intercept('**/users', { fixture: 'someUsers' });
```

Comment:
The `cy.intercept` call should be followed by `.as('someUsersRequest')` to ensure the request can be waited on or asserted later in the test.